---
title: |
  ![](ks_new_logo.svg){width=1in} 
  Kaleidoscope Report
author: "The Cognitive Disorders Research Laboratory (CDRL)"
date: "`r format(Sys.time(), '%d %B, %Y - %H:%M:%S %Z')`"
output: 
  html_document:
    toc: true
    toc_depth: 2
    toc_float: true
    code_folding: hide
    theme: paper
  pdf_document:
    toc: true
    toc_depth: 2
    number_sections: true
    fig_caption: true
    fig_height: 3
    fig_width: 5
    keep_tex: true
    template: null
    includes:
      in_header: header.tex
params:
  gene: NA
  brainrnaseq: True
  ilincs: True
  gwas: True
  braincloud: True
  string: True
  brainatlas: True
  gtex: True
  
---

<script src="string_embedded_network_v2.0.2.js"></script>

```{r setup, include=FALSE}



# htmltools::tagList(
#   htmltools::tags$script(src = "string_embedded_network_v2.0.2.js")
# )

# This file contains CSS, HTML, and R code.
# The following major sections are included:
# 1. About
# 2. Input
# 3. Output
# 3.1. Human
# 3.2. Mouse
# 3.3. KD and OE Signatures (ILINCS Info)
# 3.4. Cell Type
# 3.5. Table
# 3.6. String Info
# 3.7. GWAS Info
# 3.8. Brain Cloud Info
# 3.9. Brain Atlas Info
# 3.10. GTEx Info

# In order to make this code better, I must do the following:
# 1. Make the code more modular
# 2. Make the code more readable
# 3. Make the code more efficient

# I will start by making the code more modular. I will create a function for each section of the code.

# I will start by creating a function for the about section.

knitr::opts_chunk$set(echo = TRUE, warning = F, error = F, message = F)

library(downloadthis)


```
# About

***Kaleidoscope*** is an interactive R Shiny web application that provides a platform for an easy access to biological databases and bioinformatics tools via a user-friendly interface that could be explored by researchers to test hypotheses in silico. Query your target genes across different biological databases and tools to explore your research questions. The observations obtained from this platform could supplement existing hypotheses, spawn new ones and possibly direct future studies.

# Input

```{r, echo = FALSE}
genes <- process_gene_input(params$gene)
```

**`r toupper(params$gene)`**


